wasm2c/wasm-rt.h

```diff
@@ -308,23 +332,48 @@ uint32_t wasm_rt_grow_memory(wasm_rt_memory_t*, uint32_t pages);
 void wasm_rt_free_memory(wasm_rt_memory_t*);
 
 /**
- * Initialize a Table object with an element count of `elements` and a maximum
- * page size of `max_elements`.
+ * Initialize a funcref Table object with an element count of `elements` and a
+ * maximum size of `max_elements`.
  *
  *  ```
- *    wasm_rt_table_t my_table;
- *    // 5 elemnets and a maximum of 10 elements.
- *    wasm_rt_allocate_table(&my_table, 5, 10);
+ *    wasm_rt_funcref_table_t my_table;
+ *    // 5 elements and a maximum of 10 elements.
+ *    wasm_rt_allocate_funcref_table(&my_table, 5, 10);
  *  ```
  */
-void wasm_rt_allocate_table(wasm_rt_table_t*,
-                            uint32_t elements,
-                            uint32_t max_elements);
+void wasm_rt_allocate_funcref_table(wasm_rt_funcref_table_t*,
+                                    uint32_t elements,
+                                    uint32_t max_elements);
+
+/**
+ * Free a funcref Table object.
+ */
+void wasm_rt_free_funcref_table(wasm_rt_funcref_table_t*);
+
+/**
+ * Initialize an externref Table object with an element count
+ * of `elements` and a maximum size of `max_elements`.
+ * Usage as per wasm_rt_allocate_funcref_table.
+ */
+void wasm_rt_allocate_externref_table(wasm_rt_externref_table_t*,
+                                      uint32_t elements,
+                                      uint32_t max_elements);
 
 /**
- * Free a Table object.
+ * Free an externref Table object.
  */
-void wasm_rt_free_table(wasm_rt_table_t*);
+void wasm_rt_free_externref_table(wasm_rt_externref_table_t*);
+
+/** Grow a Table object by `delta` elements (giving the new elements the value
```

Comment:
Can you put opening and closing comment marks on lines by themsevles.

Reply:
Done.